SS2-Project/Assets/Starstorm2/Modules/Pickups/Items/Tier1/ChucklingFungus.cs

```diff
@@ -39,18 +42,9 @@ public override void Initialize()
         //Should return true only if dungus is available as well, unsure how to do that lol
         public override bool IsAvailable(ContentPack contentPack)
         {
-            return false;
+            return contentPack.itemDefs.Find("DormantFungus"); // ^like this
```

Comment:
As much as id like to do this, which is simple enough, i cant find a good way to make sure that the DormantFungus itemDef is added to the ContentPack prior to the Chuckling fungus trying to get loaded. I doubt this would work and probably just using the Config check would suffice.